test/scripts/setup/betterAuthSetup.test.ts

```
@@ -0,0 +1,65 @@
+import dotenv from "dotenv";
+import inquirer from "inquirer";
+import { betterAuthSetup } from "scripts/setup/setup"; // Import your betterAuthSetup function
+import { beforeEach, describe, expect, it, vi } from "vitest";
+import { generateBetterauthSecret } from "scripts/setup/setup"; // Adjust the path as needed
+import crypto from "crypto";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)